modules/ROOT/pages/multi-tenancy-and-tenant-configuration.adoc

```diff
@@ -9,6 +9,17 @@ One single platform to administrate with multiple logical engines. Discover what
 For Enterprise and Performance editions only.
 ====
 
+[WARNING]
+====
+This feature was deprecated in Bonita 2021.1. This means that Multi-tenant architecture will continue to work in Bonita for existing Bonita subscription customers,
+but it will not be offered to new customers, nor offered as a modification for existing Bonita subscription customers who are not already using multi-tenancy.
+====
+
+[IMPORTANT]
+====
+Bonita Enterprise customers need a contractual agreement to create additional tenants. For any new tenant creation, any general or contractual question related to Multi-tenancy, please contact your customer success representative.
+====
```

Comment:
⚠️ Using too many different kinds of admonitions make the documentation hard to read.
Admonitions are here to highlight something specific, but here, everything is highlighted. Too many highlights ~ no highlights.


cc @alachambre @NathalieC 

![image](https://user-images.githubusercontent.com/27200110/143876003-cb36dd80-11e1-4522-a04c-d23abb2a4aad.png)

Reply:
I have removed the "Note" but the information in "warning" and "error" should be displayed this way.

Reply:
The information about the editions supporting the feature is no more consistent with the other pages: we  always display it in a NOTE admonition. We plan to manage it in a different way in the future and this part will be generated. This will avoid manually declare the block everywhere. So, doing it differently here will also introduce future side effects.

Having both a WARNING and an IMPORTANT block is still weird: why not putting a single IMPORTANT block?
The message you want to convey can fit in a single block
- this is deprecated and candidate for removal
- if you are still using it, talk to your CS representative

Here is what we currently see
![image](https://user-images.githubusercontent.com/27200110/144184466-4b71eca6-c36f-40a6-b387-43dba207cb60.png)

Reply:
Because that's how CS wants to have the information displaied. We had a single warning paragraph and clients said it wasnt clear. Hence the changes. I'll add back the note and that will be my last change. Thank you for understanding :)